netstandard/ref/System.Reflection.Emit.cs

```diff
@@ -370,7 +449,9 @@ public sealed partial class MethodBuilder : System.Reflection.MethodInfo
         public override System.Reflection.ParameterInfo ReturnParameter { get { throw null; } }
         public override System.Type ReturnType { get { throw null; } }
         public override System.Reflection.ICustomAttributeProvider ReturnTypeCustomAttributes { get { throw null; } }
+        public string Signature { get { throw null; } }
 //CAS   public void AddDeclarativeSecurity(System.Security.Permissions.SecurityAction action, System.Security.PermissionSet pset) { }
+        public void CreateMethodBody(byte[] il, int count) { }
```

Comment:
What is count here for?

Reply:
Read the docs: https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.methodbuilder.createmethodbody?view=netframework-4.7.2

This is poorly designed nearly useless confusing API. I would be fine with not having it. The properly designed equivalent with super set of functionality is SetMethodBody.

Reply:
I've excluded the API. FWIW, `SetMethodBody` was added for Roslyn but they moved away from RefEmit and instead use `System.Reflection.Metadata`, which is what I'd expect real-world compilers to do anyway. And if you're not building a real compiler, why would you have your own byte code generator?